app/classes/Controllers/ComponentListing.php

```diff
@@ -14,15 +14,33 @@ class ComponentListing extends BaseController {
 
 	public function get() {
 
-		$viewdata = array();
+		$viewdata = array(
+				'primitives' => array(
+					'title' => 'Primitives',
+					'modules' => array(),
+				),
+				'component' => array(
+					'title' => 'Components',
+					'modules' => array(),
+				),
+				'templates' => array(
```

Comment:
Yeh completely agree, was struggling to think of good options for the name though and I didn't want it to hold me up too much. Might be one to just throw out for ideas in the open meeting to see what people think?

It is essentially just `o-techdocs` yeh, but I think header/footer could possibly join it too... they're more like "organisms" than most of the other components...